server/webhook_parser.go

```diff
@@ -170,8 +169,6 @@ func parseWebhookChangeLog(jwh *JiraWebhook) Webhook {
 
 		if event != nil {
 			events = append(events, event)
-		} else {
-			fieldsNotFound = append(fieldsNotFound, field)
```

Comment:
Please doublecheck this change has no effects, doesn't look like it's used.

Reply:
Look good to me :+1: